Resources/public/js/views/ez-contenteditview.js

```diff
@@ -53,6 +56,18 @@ YUI.add('ez-contenteditview', function (Y) {
                 }
             });
 
+            this.after('activeChange', function (e) {
+                if ( e.newVal ) {
+                    that._attachedViewEvents.push(
```

Comment:
`this` is perfectly usable here and given the *complexity* of the code, that would deserve an external method